src/main/java/seedu/findvisor/model/person/Phone.java

```diff
@@ -11,8 +11,9 @@ public class Phone {
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers should only contain numbers, and it should be a valid Singaporean mobile number.\n"
```

Comment:
Should there be an indication that whitespace is accepted but trimmed?

Reply:
Yes, I will do another commit to fix this